pyfstat/core.py

```diff
@@ -415,6 +427,7 @@ def _get_SFTCatalog(self):
             SFTCatalog = lalpulsar.SFTCatalog()
             Toverlap = 0
             self.detector_names = self.detectors.split(",")
+            self.numDetectors = len(self.detector_names)
```

Comment:
Should we make this more robust with https://lscsoft.docs.ligo.org/lalsuite/lalpulsar/group___s_f_tutils__h.html#gafd482631e24c3af4a5458412c99cccf7 ?

Reply:
[MultiAddToFakeSFTCatalog](https://lscsoft.docs.ligo.org/lalsuite/lalpulsar/group___s_f_tutils__h.html#ga47d5e1fa610345c239ba9f0418e4048d) a few lines down already calls [XLALGetChannelPrefix](https://lscsoft.docs.ligo.org/lalsuite/lalpulsar/group___s_f_tutils__h.html#gadd0ea28e0308a22f53bbabaa3b2896be) which _should_ also fail on invalid names, so I think we _should_ be good here...?

Reply:
Looks like so